packages/terra-popup/src/_PopupOverlay.jsx

```diff
@@ -55,15 +55,20 @@ class PopupOverlay extends React.Component {
 
     return (
       <React.Fragment>
-        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+        {
+          // Usage of tabIndex="0" to allow users to focus on potentially scrollable content
+          /* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-tabindex */
+        }
         <div
           onClick={this.handleOnClick}
           {...customProps}
           className={cx(['popup-overlay', customProps.className])}
+          tabIndex="0"
```

Comment:
Should the overlay be focusable?

Reply:
See comment on https://github.com/cerner/terra-framework/pull/772#discussion_r301360461
However, I do think this is a good question, I can't think of much reason the overlay needs to be scrollable.

Reply:
The reason it's scrollable is to kill scrolls behind it on mobile.